hrms/hr/doctype/attendance/attendance_calendar.js

```diff
@@ -2,8 +2,8 @@
 // For license information, please see license.txt
 frappe.views.calendar["Attendance"] = {
 	field_map: {
-		start: "start",
-		end: "end",
+		start: "attendance_date",
+		end: "attendance_date",
```

Comment:
This will break the calendar view?

Since the backend API passes start & end and as the `attendance_date` and `attendance_date` is not explicitly passed
https://github.com/frappe/hrms/blob/0487ba25edc0a19694848d7ec5727e02c14b25a2/hrms/hr/doctype/attendance/attendance.py#L270-L271

Reply:
This issue has been present for a long time. I use my ERP hosted on Frappe Cloud and tried to update/add fields in the attendance doctype, but had no luck.

Reply:
> This will break the calendar view?

Hi @ruchamahabal,

The issue isn't breaking the calendar view; rather, it's showing a mismatch. Can you suggest what we should do about this?

When we add the `attendance_date`, the Gantt chart works, but the calendar shows a mismatch. Conversely, when we use `start` and `end`, the calendar displays correctly, but the Gantt chart does not work.